tests/unit/ember-sinon-qunit-test.js

```diff
@@ -56,3 +56,15 @@ test('sinon sandbox cleans up after itself', function (assert) {
   assert.ok(!spy.called, 'spy has no registered calls');
   assert.ok(!stub.called, 'stub has no registered calls');
 });
+
+test('does not fake timers by default', function (assert) {
+  assert.notEqual(Date.now(), 0, 'Date.now() is not reset');
+});
+
+test('does fake timers on request', { useFakeTimers: true }, function (assert) {
```

Comment:
we've been using something like this at https://github.com/jerel/ember-cli-clock/blob/master/tests/unit/services/clock-test.js, although we use lolex, which is the fake-timers lib extracted from sinon.

I tend to agree by now that maybe it's just not worth the extra complexity...